### DISCUSSION TITLE
Blank page with Hello World example

### DISCUSSION THREAD

**Discussion by Slapppy109** [idom-team/idom#1148] [Problem] [answered]
Hello, I wanted to experiment with ReactPy, but when I tried to make a simple Hello World App, copying and pasting from the examples on the website, I'm pretty much not getting anything. Is there some missing settings or did I install something incorrectly? All I did was copy paste the pip install command for all the backends as well just to check. I also tried running the hello world on different backends, with no avail. I tried on Google Chrome and Edge. I also tried to copy paste the dog photo example, which also did not work. Does anyone know what could be wrong?
![reactpy](https://github.com/reactive-python/reactpy/assets/28899655/38fb1b88-5540-476b-b327-ec2285131d48)

**Comment by Archmonger** [comment]
Do you have any errors in your browser's console?

**Reply by Slapppy109** [reply]
Actually, I do.
![image](https://github.com/reactive-python/reactpy/assets/28899655/279d897e-d22f-43bb-9094-56a0b357906e)

**Reply by Archmonger** [reply]
What backend are you using?

**Reply by Slapppy109** [reply]
I'm using FastAPI, but similar error occurs with Starlette too

**Reply by Archmonger** [reply, marked as answer]
This is a [common bug](https://github.com/reactive-python/reactpy/discussions/1073) with Windows machines, and is typically caused by Visual Studio corrupting the Windows registry.

To summarize the fix:

Go to `HKEY_CLASSES_ROOT/.js` and create a String Value key.
Name: `Content Type`
Value: `application/javascript`

**Reply by Slapppy109** [reply]
So besides doing that change, I was able to get it to work by changing the port to 7000 rather than the default 8000
